plan/plan.go

```diff
@@ -154,8 +154,10 @@ type LogicalPlan interface {
 	// how many columns referenced by inner plan exactly.
 	PruneColumnsAndResolveIndices([]*expression.Column) ([]*expression.Column, error)
 
-	// convert2PhysicalPlan converts logical plan to physical plan. The arg prop means the required sort property.
-	// This function returns the best plan that matches the required property strictly containing the info of count, cost and plan.
+	// convert2PhysicalPlan converts logical plan to physical plan.
+	// It is called recursively from parent to children to create the result physical plan.
```

Comment:
It is called recursively from the parent to the children to create the result physical plan.